sw/device/silicon_creator/rom_ext/rom_ext_common.ld

```diff
@@ -182,6 +182,9 @@ SECTIONS {
   .debug_ranges 0x0 : { *(.debug_ranges) }
   .debug_str 0x0 : { *(.debug_str) }
   .debug_frame 0x0 : { *(.debug_frame) }
+  .debug_line_str 0x0 : { *(.debug_line_str) }
```

Comment:
What are these copy directives for? It's interesting we didn't need it until now, so we should note why we need them.

Reply:
They are also in the "bare-metal" image under silicon owner.